examples/issue.rs

```diff
@@ -6,6 +6,7 @@
 async fn main() {
     let key_str = include_str!("../tests/rsa2048-2020-08-25.json");
     let key: ssi::jwk::JWK = serde_json::from_str(key_str).unwrap();
+    // key.algorithm = Some(ssi::jwk::Algorithm::PS256);
```

Comment:
Leftover comment

Reply:
Removed in e04a671bf7366831d3951ee74d3033ecc86ffb81